src/prospectScraper/ProspectScraper.cs

```diff
@@ -0,0 +1,625 @@
+        {
+            var childNodes = tableRow.ChildNodes;
+            var node1 = childNodes[1].InnerText; //pick number?
+            string pickNumber = node1.Replace("\r", "")
+                                    .Replace("\n", "")
+                                    .Replace("\t", "")
+                                    .Replace(" ", "");
+            var node3 = childNodes[3]; //team (and team image)?
+            var teamCity = node3.ChildNodes[0].InnerText
+                                    .Replace("\r", "")
+                                    .Replace("\n", "")
+                                    .Replace("\t", "")
+                                    .TrimEnd();
+            var node5 = childNodes[5]; //Has Child Nodes - Player, School, Position, Reach/Value
+            string playerName = node5.ChildNodes[1].InnerText
+                                    .Replace("\r", "")
+                                    .Replace("\n", "")
+                                    .Replace("\t", "")
+                                    .TrimEnd();
+            string playerSchoolBeforeChecking = node5.ChildNodes[3].InnerText
+                                    .Replace("\r", "")
+                                    .Replace("\n", "")
+                                    .Replace("\t", "")
+                                    .TrimEnd(); // this may have a space afterwards.
+            string playerSchool = School.CheckSchool(playerSchoolBeforeChecking);
+            string playerPosition = node5.ChildNodes[5].InnerText
+                                    .Replace("\r", "")
+                                    .Replace("\n", "")
+                                    .Replace("\t", "")
+                                    .Replace(" ", "");
+            string reachValue = node5.ChildNodes[9].InnerText
+                                    .Replace("\r", "")
+                                    .Replace("\n", "")
+                                    .Replace("\t", "")
+                                    .Replace(" ", "");
+
+            var mdp = new MockDraftPick(pickNumber, teamCity, playerName, playerSchool, playerPosition, reachValue, pickDate);
+            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Mock Draft Round: " + mdp.round + Environment.NewLine);
+            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Pick Number: " + mdp.pickNumber + Environment.NewLine);
+            File.AppendAllText($"logs{Path.DirectorySeparatorChar}Prospects.log", "Player: " + mdp.playerName + Environment.NewLine);
+            return mdp;
+        }
+
+        public static string GetDraftDate(HtmlDocument doc)
+        {
+            return FormatDraftDate(doc.DocumentNode.SelectSingleNode("//*[@id='HeadlineInfo1']").InnerText);
+        }
+
+        public static string FormatDraftDate(string headlineInfo)
```

Comment:
This method is no longer needed.

Reply:
This is referenced in the tests. I need to unfurl that at some point. We want to remove as much as possible static classes and fields, as they tightly couple the code.